go/client/cmd_chat_send.go

```diff
@@ -18,12 +18,13 @@ import (
 
 type cmdChatSend struct {
 	libkb.Contextified
-	resolver chatCLIConversationResolver
+	resolvingRequest chatConversationResolvingRequest
 	// Only one of these should be set
 	message       string
 	setTopicName  string
 	setHeadline   string
 	clearHeadline bool
+	useStdin      bool
```

Comment:
Yes for now; this is for "send", which still has the menu. I'll create new tickets for removing menu, using ConvID prefix instead of numbers, and ConvID based resolving.